en/migrations.rst

```diff
@@ -204,14 +204,17 @@ Columns definition
 When using columns in the command line, it may be handy to remember that they
 follow the following pattern::
 
-    fieldName:fieldType[length]:indexType:indexName
+    fieldName:fieldType?[length]:indexType:indexName
 
 For instance, the following are all valid ways of specifying an email field:
 
+* ``description:string?``
```

Comment:
Doesn't fit to the header "For instance, the following are all valid ways of specifying an email field:"
Better to either add a new paragraph or keep the way you had it.